### ISSUE TITLE
Glitchy Multiblock Assembler

### ISSUE THREAD

**Issue by TheLimePixel** [Lumaceon/ClockworkPhase2#13]
After the multiblock is complete, the block space where the assembler was has an invisible ghost block
![image](https://user-images.githubusercontent.com/25333164/29742367-3fef08cc-8a7e-11e7-8b9f-f5e073bb7e06.png)

 When you walk into it, you get a missing texture:
![image](https://user-images.githubusercontent.com/25333164/29742372-5babd04a-8a7e-11e7-8f9f-b2b60b31d6bc.png)

**Comment by Lumaceon** [comment]
I'm getting the ghost block, but what are those orange dots? Are they static textures or particles? Are you maybe using Optifine?

For reference, this is what it should look like, with all the textures invisible:
[http://i.imgur.com/0QVr70G.gifv](http://i.imgur.com/0QVr70G.gif)

**Comment by TheLimePixel** [comment]
No, I am not using Optifine. They don't seem like particles. It looks like an animated texture, so I thought that that was intentional. Yet, they move so fast, that I wondering if you wanted to give people a headache.

**Comment by Lumaceon** [comment]
Nope, that'd be a visual bug. If possible, I'd like a list of mods you're using and your video settings, because, at the moment, I can't reproduce it.

**Comment by TheLimePixel** [comment]
This won't be helpful: https://minecraft.curseforge.com/projects/modern-skyblock-2/pages/versions/msb2-mod-list
I mean, we could test out all the rendering core mods, but there's still quite a few of them.

**Comment by Lumaceon** [comment]
Fixed the missing texture bug, but narrowing down the mod screwing with the fluid textures will take a while. I'll revisit this later.
